benchmarking/runs/run_diversify_2_friends.py

```diff
@@ -0,0 +1,85 @@
+import math
+
+from benchmarking.data.interfaces import MockStudentProviderSettings
+from benchmarking.data.simulated_data.mock_student_provider import MockStudentProvider
+from benchmarking.evaluations.metrics.average_gini_index import AverageGiniIndex
+from benchmarking.evaluations.metrics.num_requirements_satisfied import (
+    NumRequirementsSatisfied,
+)
+from benchmarking.evaluations.metrics.num_teams_meeting_requirements import (
+    NumTeamsMeetingRequirements,
+)
+from benchmarking.evaluations.scenarios.diversify_social_min_2_friends import (
+    DiversifySocialMin2Friends,
+)
+from benchmarking.simulation.simulation import Simulation
+from models.enums import ScenarioAttribute, Relationship
+
+
+def run_diversify_2_friends():
+    """
+    Goal: Run diversify gender scenario, measure average gini index
+    """
+
+    # Defining our changing x-values (in the graph sense)
+    class_sizes = [100, 150, 200, 250, 300]
+    num_trials = 10
+    ratio_of_friends = 0.5
+    ratio_of_enemies = 0.1
+
+    for class_size in class_sizes:
+        print("CLASS SIZE /", class_size)
+
+        number_of_teams = math.ceil(class_size / 5)
+
+        # set up either mock or real data
+        student_provider_settings = MockStudentProviderSettings(
+            number_of_students=class_size,
+            number_of_friends=4,
+            number_of_enemies=1,
+            friend_distribution="cluster",
+            attribute_ranges={
+                ScenarioAttribute.SOCIAL.value: [
```

Comment:
Read later comment but tldr; this isn't how we specify friends/enemies, the `number_of_friends`/`number_of_enemies` is how we do that.

Also, every non-friend, non-enemy relationship is defaulted to be neutral in the algorithms themselves, so we don't need to mock that